src/cascadia/TerminalApp/TerminalApp.vcxproj

```diff
@@ -118,7 +110,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup>
     <Link>
-      <AdditionalDependencies>WindowsApp.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>Kernel32.lib;WindowsApp.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
The `kernel32.lib` reference makes me uncomfortable. The calls should already be accessible through one of the onecore umbrella libs, which I believe should be getting imported somewhere in here. (I thought I saw @DHowett-MSFT use those at one point...)

Which specific method is forcing you to add this reference?

Reply:
You're right, I just tried it and it wasn't required anymore.  I can't recall which function required it either.